website/docs/en/config/lib/auto-extension.mdx

```diff
@@ -3,4 +3,50 @@
 - **Type:** `boolean`
 - **Default:** `true`
 
-Whether to automatically set the file extension based on the `format` option in the JS and DTS output files.
+Whether to automatically set the file extension based on the [`format`](/config/lib/format) option in the JavaScript output files.
+
+## Default Extension
```

Comment:
```suggestion
## Default extension
```

Reply:
I think markdown heading should be capitalized

Reply:
H1 shoud, others might not. It varies, I think we can simply follow Rsbuild.

Reply:
I just talked to them, Rsbuild use capitalized headings but they do not follow it in every doc, we just need to unify the use in exact one markdown file, all capitalized or all not. For simple two words, I prefer to use capitalized headings.

Reply:
I'm OK with both, just make it unified.